sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -8,9 +8,9 @@
 		<Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
 		<AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-		<Version>2.9.0-preview</Version>
+		<Version>2.10.0-preview</Version>
 		<PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
-		<PackageReleaseNotes>Manual failover update for IotHub.</PackageReleaseNotes>
+		<PackageReleaseNotes>Regenerated IotHub Sdk.</PackageReleaseNotes>
```

Comment:
This string ends up being displayed on nuget and viewed by the customer. Please make it more informative and user-friendly. Thanks

Reply:
String updated